package.json

```diff
@@ -4,11 +4,14 @@
   "description": "Enhance your HF experience, a native browser extension.",
   "main": "src/HFX.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint \"./src/**\" \"./tests/**\"",
+    "lint": "./node_modules/.bin/eslint \"./src/**\" \"./tests/**\" --fix",
```

Comment:
This doesn't line up with the linting settings on my personal PC (which is also eslint), so I need to figure out why.

Reply:
Try setting `"root": true` in `.eslintrc.json` (located in the repos root directory) then running again.

Reply:
Didn't seem to work. My local adds the space between function and it's parenthesis while the linter wants to remove the space:
```$(".post").each(function (index) {```

Unsure which is typically done.